auxiliary/ERC4626ToAssetsRatioOracle/package.json

```diff
@@ -29,7 +29,7 @@
     "@synthetixio/docgen": "workspace:*",
     "@synthetixio/main": "workspace:*",
     "@synthetixio/wei": "^2.74.4",
-    "@usecannon/cli": "2.13.6",
+    "@usecannon/cli": "2.14.3",
```

Comment:
Let's not mix feature with cannon updates. Please submit cannon update as a separate PR

Reply:
https://github.com/Synthetixio/synthetix-v3/pull/2162